stripper/ze_dark_souls_ptd_csgo2.cfg

```diff
@@ -55,4 +55,293 @@ modify:
 	{
 		"blockdamage" "99999"
 	}
+}
+
+;Make it so that entwatch can watch item max uses
```

Comment:
Put a disclaimer here for external VScript requirements, [an example](https://github.com/gflclan-cs-go-ze/ZE-Configs/blob/0636c85ecb3fc6ef5f22ea922be638c2b04f3385/stripper/ze_dark_souls_ptd_v0_4_csgo7.cfg#L250).